### PR TITLE
Dedupes lodash and adds webpack-bundle-analyzer

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Saves 20KB of minified bundle size. 

https://github.com/medic/angular10-migration/issues/39

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Bundle sizes are now: 
```
ASS  ./build/ddocs/medic/_attachments/main.js: 736.55KB < maxSize 800KB (gzip) 
 PASS  ./build/ddocs/medic/_attachments/polyfills-es5.js: 41.82KB < maxSize 50KB (gzip) 
 PASS  ./build/ddocs/medic/_attachments/polyfills.js: 13.64KB < maxSize 15KB (gzip) 
 PASS  ./build/ddocs/medic/_attachments/runtime.js: 719B < maxSize 1KB (gzip) 
 PASS  ./build/ddocs/medic/_attachments/scripts.js: 46.79KB < maxSize 55KB (gzip) 
 PASS  ./build/ddocs/medic/_attachments/styles.css: 55.3KB < maxSize 70KB (gzip) 
```
seen here https://github.com/medic/cht-core/runs/1912162469#step:14:830
total: 891kB

Bundle sizes for `master`:
```
./build/ddocs/medic/_attachments/js/inbox.js   815.17/820kB
./build/ddocs/medic/_attachments/js/templates.js  22.18/30kB
./build/ddocs/medic/_attachments/css/inbox.css  57.05/70kB
```
total: 894 kB

**Comment by dianabarsan** [comment]
hey @garethbowen . Would you mind having a look? 
Thank you!

File: webapp/custom-webpack.config.js [hunk 1]
```diff
@@ -15,6 +16,12 @@ module.exports = {
       '../../js/dropdown.jquery': 'node_modules/bootstrap/js/dropdown',
       'bikram-sambat': 'node_modules/bikram-sambat',
       'messageformat': 'node_modules/messageformat/index',
+      'lodash/core': 'node_modules/lodash/core',
+      'lodash/uniqBy': 'node_modules/lodash/uniqBy',
+      'lodash/flatten': 'node_modules/lodash/flatten',
+      'lodash/intersection': 'node_modules/lodash/intersection',
+      'lodash/partial': 'node_modules/lodash/partial',
+      'lodash/uniq': 'node_modules/lodash/uniq',
```
Comment (garethbowen) [inline webapp/custom-webpack.config.js:24]: 1. Why doesn't tree shaking work out of the box?
2. Can we alias these to `lodash-es` to remove the dependency on `lodash`?

**Review by garethbowen — APPROVED** [review]
Approved pending a couple of minor questions inline.

File: webapp/custom-webpack.config.js (hunk 1 above)
Comment (dianabarsan) [reply webapp/custom-webpack.config.js:24]: Lodash-es doesn't have the concept of "core", that we use in shared libraries :( 

Lodash-es is not as closely maintained as lodash, so it'd be best we steer off of it completely. 
Treeshaking is, aparently, fragile business.

Comment (dianabarsan) [reply webapp/custom-webpack.config.js:24]: I can alias everything else to lodash-es, except for core.

**Pull request merged**